tsutils/user_interaction.py

```diff
@@ -22,7 +23,8 @@ async def send_repeated_consecutive_messages(ctx, message: str) -> discord.Messa
 
 async def get_user_confirmation(ctx, text: str,
                                 yes_emoji: SendableEmoji = YES_EMOJI, no_emoji: SendableEmoji = NO_EMOJI,
-                                timeout: int = 10) -> Literal[True, False, None]:
+                                timeout: int = 10, force_delete: bool = None, show_feedback: bool = False) \
```

Comment:
`force_delete: Optional[bool]` right?

Reply:
No, it's closer to Literal[True, False, None], but the input should never be None.

Reply:
yeah Optional isn't right here imo

Reply:
doesnt Optional mean that providing the param is optional

Reply:
yeah i asked max he says Optional is right